### PR TITLE
10.0 chess tournaments

### PR DESCRIPTION
**Pull request by ArtyomLosev**

### CODE REVIEW

File: chess/models/models.py [hunk 1]
```diff
@@ -400,3 +407,96 @@ def broadcast(self, message, game_id):
...
+    players = fields.Many2many('res.users', string='Players')
+    status = fields.Char(default='Active')
+    start_date = fields.Datetime(string='Start Date', default=datetime.datetime.now())
+    tournament_type = fields.Selection([('blitz', 'Blitz'), ('limited time', 'Limited time'),
+                                        ('standart', 'Standart')], 'Game Type')
+    games = fields.One2many('chess.game', 'tournament')
+    time_d = fields.Float(default=0)
+    time_h = fields.Float(default=0)
+    time_m = fields.Float(default=0)
+    time_s = fields.Float(default=0)
```
Comment (yelizariev) [inline chess/models/models.py]: What are these fields?

File: chess/models/models.py [hunk 2]
```diff
@@ -400,3 +407,96 @@ def broadcast(self, message, game_id):
...
+            games.append({'id': g.id, 'player1': g.first_user_id, 'player2:': g.second_user_id})
+        return games
+
+
+class TournamentChessGame(models.Model):
+    _inherit = ['chess.game']
+
+    tournament = fields.Many2one('chess.tournament', ondelete='cascade')
+    first_user_score = fields.Float(default=0)
+    second_user_score = fields.Float(default=0)
```
Comment (yelizariev) [inline chess/models/models.py]: Describe this fields. It's not obvious that they are used for score, which values they can have etc.

Comment (yelizariev) [inline chess/models/models.py]: use ``help=``

File: chess/models/models.py (hunk 1 above)
Comment (ArtyomLosev) [inline chess/models/models.py]: These fields are used to store time data for blitz and limited time game modes. Further they are used as pre-defined data for all games within the tournament

File: chess/models/models.py [hunk 3]
```diff
@@ -454,8 +454,8 @@ class TournamentChessGame(models.Model):
     _inherit = ['chess.game']
 
     tournament = fields.Many2one('chess.tournament', ondelete='cascade')
-    first_user_score = fields.Float(default=0)
-    second_user_score = fields.Float(default=0)
+    first_user_score = fields.Float(default=0, help='First user game result.')
+    second_user_score = fields.Float(default=0, help='Second user game result.')
```
Comment (yelizariev) [inline chess/models/models.py]: Add description of possible values as well

**Comment by yelizariev** [comment]
Tournaments are tested.

**Pull request merged**